pkg/kube/cr_manager.go

```diff
@@ -148,54 +149,56 @@ func NewCRManager(clientset kubernetes.Interface, clientsetext extapi.Apiextensi
 	}
 }
 
-func (m *crManager) Init(ctx context.Context) (err error) {
-	err = m.createOrUpdateCRD(ctx, &aquasecurityv1alpha1.VulnerabilityReportsCRD)
+func (m *crManager) Init(ctx context.Context, initOptions *InitOptions, outWriter io.Writer) (err error) {
```

Comment:
If you passed the initOptions and the output Writer as args to the NewCRManager constructor function and saved it as crManager properties you'd not need to change methods' signatures in so many places.

Reply:
Thanks for that... I was worried that I was making a lot of signature changes.